src/XModem.cpp

```diff
@@ -510,3 +520,11 @@ static void XModem::crc_16_chksum(byte *data, size_t dataSize, byte *chksum) {
     }
   }
 }
+
+void XModem::onXmodemReceive(bool(*callback)(byte *data, size_t dataSize))
```

Comment:
I don't understand why you've created a new function pointer and setter method that is baisically identical to the `process_rx_block` function pointer and `setRecieveBlockHandler` method